src/lib.rs

```diff
@@ -360,27 +360,9 @@ pub fn pari_qf_comp_to_decimal_string(pari_qf: GEN, index: usize) -> String {
     comp_str_slice.to_string()
 }
 
-//TODO: use str::parse
 pub fn decimal_string_to_bn(dec_string: String) -> BigInt {
```

Comment:
Perhaps it would be simpler to replace all calls for `decimal_string_to_bn` with calls to `str::parse`. Seem redundant to have a function only executing one line

Reply:
make points.
@omershlo  what do u think

Reply:
sorry, missed that. 
Sounds good to me.

Reply:
> sorry, missed that.
> Sounds good to me.

no problem, will do.